src/polyfill/chunks_fixed.rs

```diff
@@ -1,31 +0,0 @@
-/// Allows splitting a reference to an array type into fixed-length chunks.
-pub trait ChunksFixed<'a, Chunks>
-where
-    Chunks: 'a,
-{
-    fn chunks_fixed(self) -> Chunks;
-}
-
-/// `$unchuncked_len` must be divisible by `$chunk_len`.
-macro_rules! define_chunks_fixed {
-    ( $unchuncked_len:expr, $chunk_len:expr ) => {
-        define_chunks_fixed!($unchuncked_len, $chunk_len, $unchuncked_len / $chunk_len);
-    };
-
-    ( $unchuncked_len:expr, $chunk_len:expr, $chunked_len:expr ) => {
-        impl<'a, T> ChunksFixed<'a, &'a [[T; $chunk_len]; $chunked_len]>
-            for &'a [T; $unchuncked_len]
-        {
-            #[inline(always)]
-            fn chunks_fixed(self) -> &'a [[T; $chunk_len]; $chunked_len] {
```

Comment:
Since you've been asking about other places zerocopy might help: this method should be able to be replaced by [`transmute_ref!`](https://docs.rs/zerocopy/latest/zerocopy/macro.transmute_ref.html), at least in contexts where `T` and `Self` are concrete types.